Rex.podspec

```diff
@@ -18,8 +18,8 @@ Pod::Spec.new do |s|
   s.watchos.deployment_target = '2.0'
   s.tvos.deployment_target = '9.0'
 
-  s.source       = { :git => 'https://github.com/neilpa/Rex.git', :tag => '0.9.0-RC.2' }
-  s.dependency 'ReactiveCocoa', '4.0.0-RC.2'
+  s.source       = { :git => 'https://github.com/neilpa/Rex.git', :tag => '0.9.0' }
```

Comment:
I'm guessing you see into the future when @neilpa makes the actual tag? But on a serious note, could you so I can push a new spec to trunk? :dancers: